app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -902,13 +902,45 @@ def rbac_group_get_details(id)
   def rbac_group_right_tree(selected_nodes)
     case @sb[:active_rbac_group_tab]
     when 'rbac_customer_tags'
-      @tags_tree = TreeBuilderTags.new(:tags_tree,
-                                       :tags,
-                                       @sb,
-                                       true,
-                                       :edit    => @edit,
-                                       :filters => @filters,
-                                       :group   => @group)
+      cats = Classification.categories.find_all do |c|
+        c if c.show || !%w[folder_path_blue folder_path_yellow].include?(c.name)
+      end
+      cats.sort_by! { |t| t.description.try(:downcase) } # Get the categories, sort by description
+      cats.delete_if { |c| c.read_only? || c.entries.empty? } # Remove categories that are read only or have no entries
```

Comment:
You can combine this into the future `select` or `reject` described in my previous comment above.